.github/workflows/test.yml

```diff
@@ -65,8 +65,6 @@ jobs:
         uses: ./.github/workflows/install-circt
       - name: Check Formatting
         run: sbt scalafmtCheckAll
-      - name: Documentation
```

Comment:
this was redundant with the website build doing it so I removed it... could put it back